package/MDAnalysis/topology/ITPParser.py

```diff
@@ -590,6 +612,20 @@ def parse(self, include_dir='/usr/local/gromacs/share/gromacs/top/',
     def _pass(self, line):
         pass
 
+    def parse_atomtypes(self, line):
+        keys = ['type_bonded', 'atomic_number', 'mass', 'charge', 'p_type']
+        fields = line.split()
+        if len(fields[5]) == 1 and fields[5].isalpha():
+            values = fields[1:6]
+        elif len(fields[3]) == 1 and fields[3].isalpha():
```

Comment:
Could you link me to where these different specs are? The only one I see in the manual and in my own files is the one that follows the heading `;name  at.num   mass      charge  ptype       c6           c12`. Given that the lynchpin of this parsing is the particle type, is it always guaranteed to be there?

Reply:
I took them directly from the preprocessing code in gromacs, I guess that makes them as correct as possible.

https://github.com/gromacs/gromacs/blob/6dbf35dddb95932e6e98d8e923f4ba87bf257b8e/src/gromacs/gmxpreprocess/toppush.cpp#L324-L352

Reply:
Yes, yes it does -- did not think to look there!